CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 # ref: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
-* @shashankram @snehachhabria @nojnhuh @draychev @michelleN @eduser25
+* @shashankram @snehachhabria @nojnhuh @draychev @eduser25
```

Comment:
@michelleN thanks for all your efforts.

To reflect the contribution of prior maintainers, I think it would be nice to add the following to this file:
```suggestion
* @shashankram @snehachhabria @nojnhuh @draychev @eduser25

# Emeritus maintainers
# @michelleN
```